homeassistant/components/sleepiq/coordinator.py

```diff
@@ -1,40 +0,0 @@
-"""Coordinator for SleepIQ."""
-from datetime import timedelta
-import logging
-
-from sleepyq import Sleepyq
-
-from homeassistant.core import HomeAssistant
-from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
-
-from .const import BED
-
-_LOGGER = logging.getLogger(__name__)
-
-UPDATE_INTERVAL = timedelta(seconds=60)
-
-
-class SleepIQDataUpdateCoordinator(DataUpdateCoordinator[dict[str, dict]]):
```

Comment:
Please keep the existing coordinator as it makes strict typing easier when we aren't storing it in a `dict`